swig/lua/libsysrepoLua51.i

```diff
@@ -0,0 +1,77 @@
+%module libsysrepoLua51
+
+/* Filter out 'Setting a const char * variable may leak memory' warnings */
+
+%{
+/* Includes the header in the wrapper code */
+#include "Connection.h"
+#include "Session.h"
+#include "Sysrepo.h"
+
+%}
+
+/* Parse the header file to generate wrappers */
+%include "Connection.h"
+%include "Session.h"
+%include "Sysrepo.h"
+
+typedef enum sr_type_e {
```

Comment:
Just a question: Is there any way to avoid the duplication of these enums, already defined in sysrepo.h? With such a duplication, the maintainability of the code drops and we risk that someone who will update it in sysrepo.h forgets to update it here.